modules/ROOT/pages/mule-app-properties-to-configure.adoc

```diff
@@ -81,6 +81,73 @@ image::mruntime-properties-file.png[properties-file]
 </http:request-config>
 ----
 
+When the application is deployed, the property values will be computed from the different possible sources and will statically replace the placeholders as if they have been written directly in the source code.
+
+Following the previous example, it would be equivalent to have written it like this:
```

Comment:
```suggestion
The following is another example on how to configure your Global HTTP Request configuration.
```

Reply:
Mmm... this is not quite what I wanted to say. I want to show how the config would look like after property placeholder replacements while emphasizing the fact that it would be no different from having hard-coded the values directly in the source code (without using the placeholders at all).